### DISCUSSION TITLE
Key assignments

### DISCUSSION THREAD

**Discussion by gwww** [wezterm/wezterm#658] [Q&A]
If I map a key, say the Enter key, to `EmitEvent` can I set it back to default binding? If not, could this be added?

**Comment by wez** [comment]
I don't quite follow what you're asking; can you expand on your idea a bit more?

**Reply by gwww** [reply]
Say I did this:
```
return {keys={ {key='Enter', mods='', action={EmitEvent="foo-event"}}}}
```
How would I map it back to `Enter` again? Does:
```
return {keys={ {key='Enter', mods='', action=nil}}}
```
delete the binding? The delete would be done somewhere during handling an event.

As I've been thinking about this, I'm not sure I even want to do it anymore. Mostly because a key binding affects all windows/panes and I was thinking of it affecting a single pane. Regardless, is it possible to map a key back to its default?

**Reply by wez** [reply]
I think what you're asking is similar to this: https://github.com/wez/wezterm/issues/656#issuecomment-814376452
